conan/cli/formatters/report/diff_html.py

```diff
@@ -96,10 +110,12 @@
                 <h2>File list:</h2>
                 <input type="text" id="search-include" placeholder="Include search..." oninput="onIncludeSearchInput(event)" />
                 <input type="text" id="search-exclude" placeholder="Exclude search..." oninput="onExcludeSearchInput(event)" />
+                <span id="searching_icon" style="display:none">⏳</span>
```

Comment:
```suggestion
                <span id="searching_icon" style="display:none">...</span>
```
The emoji breaks the windows console